python/cuml/cuml/linear_model/ridge.pyx

```diff
@@ -208,16 +208,16 @@ class Ridge(UniversalBase,
     }
 
     @device_interop_preparation
-    def __init__(self, *, alpha=1.0, solver='eig', fit_intercept=True,
+    def __init__(self, *, alpha=1.0, solver='auto', fit_intercept=True,
                  normalize=False, handle=None, output_type=None,
                  verbose=False):
         """
         Initializes the linear ridge regression class.
 
         Parameters
         ----------
-        solver : Type: string. 'eig' (default) and 'svd' are supported
-        algorithms.
+        solver : Type: string. 'auto' (default), 'eig', 'svd', and 'cd' are supported
+        algorithms. The 'auto' option will use 'eig' as the solver.
```

Comment:
Why introduce "auto" if it is a synonym for "eig"?

Reply:
To increase parity with scikit-learn and also – why not? We may or may not always fall back to the "eigen" solver in future releases.

Reply:
Was wondering if I was missing something. Fine with me.

From our last conversation about "is cuml compatible with scikit-learn?" I took that something like this would not increase (or decrease) our compatibility "score". Because cuml itself doesn't have to support all/the same hyper-parameters and values to be compatible. Which means this isn't a change that, strictly speaking, is needed. And when I see something that I think isn't strictly needed in a diff I start to wonder if I missed something, or if it really is just a nice to have. Thanks for the discussion on this

Reply:
It's a valid question and I my response may not have given it full credit. But I really didn't have a better answer. :D